### PR TITLE
THU/more_cypress_tests_qa_2.8

### PR DESCRIPTION
**Pull request by csm-thu**

### CODE REVIEW

File: cypress/commons/actions/generic/ScenarioParameters.js [hunk 1]
```diff
@@ -69,6 +72,18 @@ function discard() {
   getParametersDiscardButton().click();
   getParametersConfirmDiscardButton().click();
 }
+
+function launch(dontAskAgain = false) {
```
Comment (nborde-CSM) [inline cypress/commons/actions/generic/ScenarioParameters.js:76]: Is not there a way to factorize launch and launchAndUpdate functions who are barely identical ?

**Review by nborde-CSM — APPROVED** [review]
One remark but LGTM :+1:

**Review by esasova — APPROVED** [review]
LGTM :+1:

File: cypress/commons/actions/generic/ScenarioParameters.js (hunk 1 above)
Comment (csm-thu) [reply cypress/commons/actions/generic/ScenarioParameters.js:76]: Done :+1:

**Pull request merged**